Api/Endpoints/Webhooks.php

```diff
@@ -0,0 +1,181 @@
+<?php
+
+namespace Lokalise\Endpoints;
+
+use \Lokalise\LokaliseApiResponse;
+use \Lokalise\Exceptions\LokaliseApiException;
+use \Lokalise\Exceptions\LokaliseResponseException;
+
+/**
+ * Class Webhooks
+ * @package Lokalise\Endpoints
+ * @link https://lokalise.co/api2docs/php/#resource-webhooks
```

Comment:
We have recently updated these broken links to correct ones https://app.lokalise.com/api2docs/curl/#...